README.md

```diff
@@ -85,60 +63,44 @@ With the supplied tomcat configuration, the following should all be mounted on v
 * `/usr/local/tomcat/conf/Catalina`
 * `/usr/local/tomcat/temp`
 
-### Parent Image
+### Disabling Security Manager
 
-Web Applications, Tomcat configuration, etc. can all be supplied by various  methods.
-We recommend using this as a [parent image](https://docs.docker.com/glossary/?term=parent%20image),
-and then following the  recommended practices for passing configuration and secrets for your orchestrator and use case.
-
-For reference, see the documentation on
-[layers](https://docs.docker.com/storage/storagedriver/#container-and-layers),
-the
-[VOLUME](https://docs.docker.com/engine/reference/builder/#volume)
-instruction,
-[best practices with VOLUMEs](https://docs.docker.com/develop/develop-images/dockerfile_best-practices/#volume),
-[best practices with persistence](https://docs.docker.com/develop/dev-best-practices/#where-and-how-to-persist-application-data),
-and
-[tmpfs](https://docs.docker.com/storage/tmpfs/) mounts.
-
-### Examples of usage as a parent image
-
-Example from a Dockerfile using a public parent image in Docker Hub.
+The Tomcat in this image is running with Security Manager switched on. This may impact performance. The Security Manager can be disabled by overriding the startup command to:
 
```

Comment:
I don't think we should document how to reduce security.

Reply:
security manager is deprecated and can be removed, maybe in another PR https://openjdk.java.net/jeps/411